.changes/next-release/deprecation-8a9cd1d2-9051-4e01-b94d-c0f52687f411.json

```diff
@@ -0,0 +1,4 @@
+{
+  "type" : "deprecation",
+  "description" : "An upcoming release will remove support for the 2020.2 family of IDEs"
```

Comment:
is "family" the right word?

Reply:
Not sure,

Maybe "for IDEs based on 2020.2"?